src/WebJobs.Script/Rpc/Configuration/WorkerConfigFactory.cs

```diff
@@ -20,19 +20,28 @@ public WorkerConfigFactory(IConfiguration config, ILogger logger)
         {
             _config = config;
             _logger = logger;
+            WorkerDirPath = Path.Combine(Path.GetDirectoryName(new Uri(typeof(WorkerConfigFactory).Assembly.CodeBase).LocalPath), LanguageWorkerConstants.DefaultWorkersDirectoryName);
+            var workersDirectorySection = _config.GetSection($"{LanguageWorkerConstants.LanguageWorkerSectionName}:{LanguageWorkerConstants.WorkersDirectorySectionName}");
+            if (!string.IsNullOrEmpty(workersDirectorySection.Value))
+            {
+                 WorkerDirPath = workersDirectorySection.Value;
+            }
         }
 
+        public string WorkerDirPath { get; }
```

Comment:
DirPath is redundant - perhaps just "WorkerDirectory"?

Reply:
done!